### PR TITLE
frisky.fm connector

### PR DESCRIPTION
**Pull request by jbwharris**
New connector for frisky.fm

**Describe the changes you made**
Adding new connector as requested in #3330. If you're using the site with only a free account you will only be able to test using the 4 radio streams. If someone with a frisky.fm account could test this more thoroughly it would be appreciated. I'm not sure if @pedrojvaz has a paid account to test.

**Additional context**
image

### CODE REVIEW

**Comment by hornbuckle** [comment]
@jbwharris Just wanted to point you in the direction of this recently updated connector for how to write selectors to leave off those last 5 random characters on the classes using attribute selectors: https://github.com/web-scrobbler/web-scrobbler/blob/master/src/connectors/epidemicsound.js

**Comment by jbwharris** [comment]
@hornbuckle Great suggestion, will do that moving forward. I just updated the connector to use a wildcard selector.

**Comment by hornbuckle** [comment]
BTW I'm not familiar with this site, but it seems as if the label should be "FRISKY" since that is how it is consistently typed. Maybe the original requester can confirm.

**Comment by jbwharris** [comment]
@pedrojvaz  that is the only site that's currently supported. I can look at the old site, but I figured if they're moving towards a new design and URL that I'd just create a connector for the new site since it won't have an expiry date.

File: src/core/connectors.js [hunk 1]
```diff
@@ -2237,6 +2237,14 @@ const connectors = [{
 	],
 	js: 'connectors/xpn.js',
 	id: 'xpn',
+}, {
+	label: 'frisky.fm',
```
Comment (inverse) [inline src/core/connectors.js:2241]:
```suggestion
	label: 'FRISKY',
```

**Review by inverse — APPROVED** [review]

**Pull request merged**